textpattern/setup/txpsql.php

```diff
@@ -125,7 +127,17 @@
     $xml = simplexml_load_string($data, "SimpleXMLElement", LIBXML_NOCDATA);
     foreach ($xml->article as $a) {
         $article = array();
-        $article['Section']   = empty($a->section) ? $section : $a->section;
+        $article['section']   = $section;
+        $article['status'] = 4;
```

Comment:
Can you use `STATUS_LIVE` here, please? And anywhere else where status codes have used magic numbers. Thanks.